sphinx/source/bokeh_theme/layout.html

```diff
@@ -28,32 +28,55 @@
   –––––––––––––––––––––––––––––––––––––––––––––––––– -->
   <link rel="stylesheet" href="{{ pathto('_static/' + theme_css, 1) }}">
 
+{%- macro css() %}
+    <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
+    <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
+    {%- for css in css_files %}
+      {%- if css|attr("filename") %}
+    {{ css_tag(css) }}
+      {%- else %}
+    <link rel="stylesheet" href="{{ pathto(css, 1) }}" type="text/css" />
+      {%- endif %}
+    {%- endfor %}
+{%- endmacro %}
+
+{%- block css %}
+{{- css() }}
+{%- endblock %}
+
   <!-- Scripts
   –––––––––––––––––––––––––––––––––––––––––––––––––– -->
 
-  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+{%- macro script() %}
+  <script type="text/javascript" id="documentation_options" data-url_root="{{ pathto('', 1) }}" src="{{ pathto('_static/documentation_options.js', 1) }}"></script>
+  {%- for js in script_files %}
+  {{ js_tag(js) }}
+  {%- endfor %}
+{%- endmacro %}
+
+{%- block scripts %}
   <script type="text/javascript">
-    var _BOKEH_CURRENT_VERSION = "{{ VERSION }}";
+   var _BOKEH_CURRENT_VERSION = "{{ VERSION }}";
   </script>
+
+  {{- script() }}
+
   <script src="{{ pathto('_static/' + theme_js, 1) }}"></script>
+{%- endblock %}
 
   <!-- Favicon
   –––––––––––––––––––––––––––––––––––––––––––––––––– -->
   <link rel="icon" type="image/png" href="{{ pathto('_static/' + theme_favicon, 1) }}">
 
-  <!-- Canonical
-  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-  <link rel="canonical" href="https://bokeh.pydata.org/en/latest/{{ pagename }}{{ file_suffix }}" />
```

Comment:
Should there still be cannonical links, but pointing to `docs.python.org` ?

Reply:
No.

The canonical link is removed from here because it automatically added by Read the Docs.

In the current online (temporal) version, Read the Docs adds:

```html
<link rel="canonical" href="https://bokeh-bokeh.readthedocs-hosted.com/en/latest/" />
```

Once the real domain is setup on Read the Docs, it will point to the domain that you setup. There more docs on this at https://docs.readthedocs.io/en/latest/guides/canonical.html